src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the literal '40' with a constant to represent the daily rental cost. This will make your code more maintainable and align with checklist item 2.